corporate/templates/atlassian.html

```diff
@@ -11,8 +11,6 @@
 
 {% block portico_content %}
 
-{% include 'zerver/landing_nav.html' %}
```

Comment:
What's the story with removing `landing_nav` here?

Reply:
It's now using the navigation from the parent template (which is overriden in ` corporate/templates/zerver/portico-header.html`) ... I'll shortly force-push a fix for the navigations that are currently broken.

Reply:
I completed this pull request and improved the commit message to explain the deal with `landing_nav`.